pkg/tinydtls/Makefile

```diff
@@ -5,6 +5,7 @@ PKG_VERSION=eb6f017ab451bb6cc4428b3e449955a76aeeba19
 PKG_LICENSE=EPL-1.0,EDL-1.0
 
 CFLAGS += -Wno-implicit-fallthrough
+CFLAGS += -D_XOPEN_SOURCE=600
```

Comment:
Please add the mailing list reference and a short notice about which newlib commit fixes it here as a comment, to explain why we need this. It will let us remove this change in the future when every toolchain is updated to newer newlib.